os_helper.bash

```diff
@@ -139,13 +139,11 @@ tRHSubscribeAttach() {
 tRHEnableEPEL() {
   tIsRHEL || skip "Not required"
   if tIsRHEL 7; then
```

Comment:
Maybe just use `tSetOSVersion` and then `${OS_VERSION}` in the URLs, get rid of these conditionals.